chainercv/transforms/image/resize.py

```diff
@@ -21,15 +39,8 @@ def resize(img, output_shape):
         ~numpy.ndarray: A resize array in CHW format.
```

Comment:
Please add a documentation about modules used depending on the environment (CV2, PIL).